internal/services/sql/sql_administrator_resource.go

```diff
@@ -25,6 +26,13 @@ func resourceSqlAdministrator() *pluginsdk.Resource {
 		Update: resourceSqlActiveDirectoryAdministratorCreateUpdate,
 		Delete: resourceSqlActiveDirectoryAdministratorDelete,
 
+		DeprecationMessage: func() string {
+			if features.ThreePointOhBeta() {
+				return "The `azurerm_sql_active_directory_administrator` resource is deprecated and will be removed in version 4.0 of the AzureRM provider. Please use the `azuread_administrator` block of the `azurerm_mssql_server` resource instead."
+			}
+			return ""
+		}(),
```

Comment:
FYI we have a helper for this:

`features.DeprecatedInThreePointOh("message")` (and also `features.DeprecatedInFourPointOh("message")`)

Reply:
Thanks, have updated 👍